### PR TITLE
Hide dashcard parameter mappings for action dashboard cards

### PR DESCRIPTION
**Pull request by iethree**
background: https://metaboat.slack.com/archives/C04FG88HL95/p1677688426691679

### Description

Since we have a snazzy modal for mapping dashboard parameters to actions, we can hide the parameter mapping interface on dashcards, especially given that action dashcards are often very tiny and the mapping component doesn't fit very well.

Before | After
--- | ---
Screen Shot 2023-03-01 at 10 03 08 PM | Screen Shot 2023-03-01 at 8 39 15 PM

### How to verify

- Create a dashboard with one or more filters. 
- Add an action with one or more parameters
- While editing the dashboard click on the filters to go into "mapping mode", you should see a nice little info icon instead of a parameter mapping input

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx
Comment (EmmadUsmani) [inline frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx:15]: Nit: this can be inline like the other props

**Review by EmmadUsmani — COMMENTED** [review]
Looks good! I will let someone else with more context on dashboard approve

File: frontend/src/metabase-types/api/mocks/metadata.ts
Comment (kulyk) [inline frontend/src/metabase-types/api/mocks/metadata.ts:16]: `metabase-types` shouldn't import stuff from `metabase-lib`. Can we move this to `metabase-lib/mocks` for now maybe? Or maybe wait until #28836 is merged and use it instead

File: frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx
Comment (kulyk) [inline frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx:35]: Is there anything else except the icon we can look up for? If the test fails, having e.g. a user-facing string would make it much easier to find a broken piece of code

**Review by kulyk — CHANGES_REQUESTED** [review]
Looking good! Requesting changes as we need to deal with where we keep the metadata mock first

File: frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx
Comment (iethree) [reply frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx:35]: I can update the component to expose an aria label. 👍

File: frontend/src/metabase-types/api/mocks/metadata.ts
Comment (kulyk) [reply frontend/src/metabase-types/api/mocks/metadata.ts:16]: FWIW #28836 is merged and IMO we should try to use it here instead

**Review by kulyk — APPROVED** [review]

**Pull request merged**
